tools/harness-thci/OpenThread.py

```diff
@@ -1498,7 +1498,7 @@ def setDefaultValues(self):
                                           ModuleHelper.Default_NwkName)
         self.pskc = hex(stretchedPSKc).rstrip('L').lstrip('0x')
         self.securityPolicySecs = ModuleHelper.Default_SecurityPolicy
-        self.securityPolicyFlags = 'onrcb'
+        self.securityPolicyFlags = 'onrc'
```

Comment:
There is some securitypolicy related THCI workaround for DEV-2173 @karthick-grl for awareness